packages/integration-karma/test/swapping/templates/index.spec.js

```diff
@@ -0,0 +1,36 @@
+import { createElement } from 'lwc';
```

Comment:
Let's also add tests: 
- for components
- for stylesheets swapping for both synthetic shadow and native shadow.

Reply:
in the making